test/common/test-functions.js

```diff
@@ -309,3 +309,7 @@ function ph_element_clip(sel) {
     var f = currentFrameAbsolutePosition();
     return { x: r.x + f.x, y: r.y + f.y, width: r.width, height: r.height, scale: 1 };
 }
+
+function ph_wait_for_animations_to_finish(sel) {
+    return Promise.all(ph_find(sel).getAnimations({subtree:true}).map(a => a.finish()));
```

Comment:
Cool about `subtree`! I was already afraid that we had to walk the tree ourselves.

[docs](https://developer.mozilla.org/en-US/docs/Web/API/element/getAnimations)